### ISSUE TITLE
(WCAG 2.1) "Choose File" Control is not keyboard accessible. 

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#5523]
OBSERVED RESULTS:

- The "Choose File" control for file upload is not keyboard accessible and lacks operability.
- The control does not have a clear role of a button or link, which can confuse users relying on keyboard navigation.
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/bc58b00e-392a-4d4c-bce5-cf39ffe82fe8)

EXPECTED RESULTS:

- The "Choose File" control should be made keyboard accessible and operable, allowing users to interact with it using keyboard input.
- Add a valid role attribute to the "Choose File" control to indicate its purpose. Consider using <button> <a> or role="button" and tabindex=”0” to convey that it's a button for selecting files.

STEPS TO REPRODUCE:

- Navigate to [URL](https://demo.civiform.dev/)
- Navigate to and activate the ‘Apply’ for the Comprehensive sample program.
- Navigate to and activate “What is your address?”.
- Fillup the form fields and continue till step 8.
- On the step 8 Upload anything from your computer, navigate to the "Choose File" control using TAB and OR SHIFT tab key.
- Observe that the "Choose File" Control is not keyboard accessible.

ENVIRONMENT DETAILS:

- Device(s) (brand, OS version, test key or release key): Windows 10 Enterprise (64-bit),MacOS Ventura Version 13.4
- Screen reader name and version: JAWS 2022.2211.7, NVDA 2022.3, VoiceOver
- Browser name and version: Chrome Version 113.0.5672.93
